.github/workflows/needs-review.yml

```diff
@@ -0,0 +1,22 @@
+name: "Needs Review Label Actions"
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    name: Comment on Needs Review label
+    if: github.event.label.name == '[Status] Needs Review'
+    steps:
+      - name: Comment
+        uses: actions/github-script@0.8.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.createComment({
```

Comment:
How cool would it be to have something like `jetpack.packages.updateMaster( { packageName } )`?